src/collective/jsonmigrator/blueprints/datafields.py

```diff
@@ -27,8 +29,15 @@ def __init__(self, transmogrifier, name, options, previous):
         self.datafield_prefix = options.get("datafield-prefix", "_datafield_")
         self.root_path_length = len(self.context.getPhysicalPath())
 
+        if "path-key" in options:
+            pathkeys = options["path-key"].splitlines()
+        else:
+            pathkeys = defaultKeys(options["blueprint"], name, "path")
+        self.pathkey = Matcher(*pathkeys)
+
```

Comment:
Was this supposed to go in here? This does not have anything to do with code analysis.

Reply:
This code was needed to define `pathkey` variable. See:

https://github.com/collective/collective.jsonmigrator/pull/42#discussion_r707636045